### PR TITLE
Snyk fix 4c124a8bdbb72372d7199391e9906158

### PR DESCRIPTION
**Pull request by patooworld**

### CODE REVIEW

**Review by patooworld — COMMENTED** [review]
reviewed

**Review by dipesh-rawat — COMMENTED** [review]
@patooworld Thanks you for your contribution. It would be helpful if you could provide details and clarification on the reasoning behind this change in the [PR description (here)](https://github.com/kubernetes/website/pull/44342#issue-2040625259). Without that information, it becomes challenging to provide any feedback.

File: .github/workflows/static.yml
Comment (dipesh-rawat) [inline .github/workflows/static.yml:1]: In this project, we typically opt not to utilize GitHub Actions.

/hold

**Review by divya-mohan0209 — COMMENTED** [review]
Hello @patooworld , good day!

This is the second instance of you making the same pull request to our repository despite conveying that we do not leverage GitHub actions and that we require you to describe the changes being made. 

I will be closing this PR. In case you want to suggest changes to the docs, we request you adhere to the process & not submit dummy PRs for the sake of it. 

/close

**Review by patooworld — COMMENTED** [review]
Reviewed

**Review by patooworld — COMMENTED** [review]
Reviewed

**Pull request closed without merging**